packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -56,12 +60,12 @@ export const DatePicker = (inProps: DatePickerProps) => {
             required={required}
             endAdornment={
                 !required ? (
-                    <>
+                    <InputAdornment position="end">
```

Comment:
Optional: You could move the component around the required if, then it would only be added once

Reply:
I had been thinking about that. That would render an empty `InputAdornment` when no `endAdornment` is set. That is why i decided against that. Are empty tags ok in a setting like this?

Reply:
No, haven't thought about that. Your solution is better 👍🏼